pkg/restapi/issuer/operation/operations_test.go

```diff
@@ -403,3 +432,36 @@ func serveHTTPMux(t *testing.T, handler Handler, endpoint string, reqBytes []byt
 
 	return rr
 }
+
+const vc = `{
```

Comment:
just curious, don't we need threadID here?

And it is safe to have check on `"connectionState": "completed"`: wallet will wait for 10 seconds for connection to complete in requested state. And will send response as state "requested" state after timeout